examples/Hyperparams And Distributions.ipynb

```diff
@@ -0,0 +1,542 @@
+{
```

Comment:
The table of content would be too much about hyperparameters. 

We already have two notebooks about hyperparams and they were already too similar. 

Please update the old notebooks to add the information there instead of creating a new one.